tests/modules/ctc/ctc_tests_ui.cc

```diff
@@ -0,0 +1,89 @@
+/*****************************************************************************
+ * @file ctc_tests.cc
+ *
+ * @brief Short-term testing for the CTC backend.
+ *****************************************************************************/
+
+#include "ctc.h"
+#include "ctc_ui.h"
+#include "csv_parser.h"
+
+#ifdef WIN32
+#include "windows.h"
+#include "commdlg.h"
+#include "block.h"
+#endif
+
+std::string ExtractFileName(const std::string& full_path) {
```

Comment:
When this eventually gets moved to somewhere in `src`, we'll want a single return here, but I don't want to hold up this PR any longer so it's ok for now